tests/connectors/kubernetes_test.py

```diff
@@ -2397,7 +2400,8 @@ async def test_create_rollout_tuning(self, _rollout_tuning_config: KubernetesCon
         if tuning_endpoint is None:
             raise AssertionError(f"Tuning pod {tuning_name} not contained in service endpoints: {endpoints}")
 
-    async def test_adjust_rollout_tuning_cpu_with_settlement(self, _rollout_tuning_config, namespace):
+    async def test_adjust_rol_tuning_cpu_with_settle(self, _rollout_tuning_config, namespace):
+        # test_adjust_rollout_tuning_cpu_with_settlement
```

Comment:
seems this line could probably go if you're cleaning up things.

Reply:
This line documents the unabbreviated test name in case the abbreviated name causes confusion.

The only reason I have to change the name is to prevent collisions in the kubetest generated namespaces which cause errors unrelated to the servox code during test setup and teardown

Reply:
That makes sense. LGTM